app/src/main/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenter.java

```diff
@@ -19,11 +23,17 @@
         super(view, configuration);
     }
 
+    private List<ScoreWeight> createScoreWeights() {
```

Comment:
Also it looks like the last score weight is being cut off by the "Let's Get Started" button